tests/hikari/test_channels.py

```diff
@@ -383,6 +383,61 @@ async def test_edit(self, model):
         )
 
 
+class TestGuildThreadChannel:
+    @pytest.fixture()
+    def model(self, mock_app):
+        return hikari_test_helpers.mock_class_namespace(channels.GuildThreadChannel)(
+            app=mock_app,
+            id=snowflakes.Snowflake(69420),
+            name="foo2",
+            type=channels.ChannelType.GUILD_PUBLIC_THREAD,
+            guild_id=snowflakes.Snowflake(123456789),
+            parent_id=snowflakes.Snowflake(987654321),
+            is_archived=False,
+            auto_archive_duration=679,
+            is_locked=False,
+            owner_id=snowflakes.Snowflake(123039302),
+            thread_created_at=234,
+            last_message_id=snowflakes.Snowflake(2345342),
+            last_pin_timestamp=4457867,
+            rate_limit_per_user=60,
+            approximate_message_count=10,
+            approximate_member_count=2,
+            archive_timestamp=0,
+            member=None
+        )
+
+    @pytest.mark.asyncio()
+    async def test_thread_edit(self, model):
```

Comment:
```suggestion
    async def test_edit(self, model):
```

Reply:
I've removed that test as it is no longer necessary, given I'm merging `GuildThreadChannel.edit` into `GuildChannel.edit`.